src/reducers/categorySlice.js

```diff
@@ -35,6 +61,28 @@ const categorySlice = createSlice({
       .addCase(getCategories.rejected, (state, action) => {
         state.loading = false;
         toast.error(action.payload);
+      })
+      .addCase(getQuizesInCategory.pending, (state) => {
+        state.loading = true;
+      })
+      .addCase(getQuizesInCategory.fulfilled, (state, action) => {
+        state.loading = false;
+        state.quizes = action.payload;
+      })
+      .addCase(getQuizesInCategory.rejected, (state, action) => {
+        state.loading = false;
+        toast.error(action.payload);
+      })
+      .addCase(getQuiz.pending, (state) => {
+        state.loading = true;
+      })
+      .addCase(getQuiz.fulfilled, (state, action) => {
+        state.loading = false;
+        state.quiz = action.payload;
+      })
+      .addCase(getQuiz.rejected, (state, action) => {
+        state.loading = false;
+        toast.error(action.payload);
```

Comment:
its great that you are using the .rejected for toast errors

Reply:
Really nicely done